app/controllers/unattended_controller.rb

```diff
@@ -11,14 +11,16 @@ class UnattendedController < ApplicationController
   # We want to find out our requesting host
   before_action :get_host_details, :except => [:hostgroup_template, :built, :failed]
   before_action :get_built_host_details, :only => [:built, :failed]
-  before_action :allowed_to_install?, :except => :hostgroup_template
+  before_action :allowed_to_install?, :except => :hostgroup_template, :unless => -> { ipxe_request? }
```

Comment:
This is hacky, can you move the logic into `allowed_to_install?` method itself?

Also, this opens doors to all iPXE requests while we only want to allow the initial request. Can we identify the initial (bootstrap) request? How about passing an extra option from DHCP filename option that would do this? Something like `/unattended/iPXE?bootstrap=yes` or we can reuse `token=boostrap` perhaps.

Reply:
Yeah, this is indeed hacky. I just ported the plugin code, still need to clean this up.

I don't get what you want to achieve with this bootstrap parameter. What's the aim?
I want to support dumb DHCP servers with this that do not support any conditional logic or different options for specific hosts.

Reply:
Let me explain, and maybe I am missing your iPXE use case. But in deployments I've worked with, because DHCP is dumb you have to boot what I call "bootstrap ipxe" script. That's a URL that never changes and contains static ipxe script. In that script, you chainboot another ipxe script but now you can use scripting capabilities therefore you can say `/unattended/iPXE?mac=${net0/mac}`.

I am under impression that the global template is actually what I call the bootstrap. I was thinking creating some extra parameter saying "get me the global template" would allow us doing cleaner code. But it can also be as simple as without any `mac` or `token` or anaconda MAC HTTP header  just return the global template. That's also an option.

Sorry I often think loud and this confuses people, I need to stop.

Reply:
It's alright. You actually make a valid point.
In my use case I currently have an embedded iPXE template that loads the global template from Foreman.

```
#!ipxe
dhcp
chain http://tfmdemo.beispiel.xyz/unattended/iPXE?mac=${net0/mac}
```

You are right, we need this as well. But I think we should add this as a separate controller action (in a separate commit).